google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseTest.java

```diff
@@ -79,6 +87,18 @@ public void setUp() {
         .thenAnswer(
             invocation ->
                 new Database.Builder(dbClient, (DatabaseId) invocation.getArguments()[0]));
+    try {
+      protoDescriptorsInputStream =
+          getClass()
+              .getClassLoader()
+              .getResourceAsStream("com/google/cloud/spanner/descriptors.pb");
```

Comment:
We should have this resource path as a constant `PROTO_DESCRIPTORS_RESOURCE_PATH`, and use it wherever needed.

Reply:
Done.